src/app/components/BibPage/BibPage.jsx

```diff
@@ -15,117 +16,149 @@ import {
   }
-  // `linkable` means that those values are links inside the app.
-  // `selfLinkable` means that those values are external links and should be self-linked,
-  // e.g. the prefLabel is the label and the URL is the id.
-  const topFields = [
-    { label: 'Title', value: 'titleDisplay', linkable: true },
-    { label: 'Author', value: 'creatorLiteral', linkable: true },
-    { label: 'Additional Authors', value: 'contributorLiteral', linkable: true },
-  ];
-  const bottomFields = [
-    { label: 'Publication', value: 'React Component' },
-    { label: 'Electronic Resource', value: 'React Component' },
-    { label: 'Description', value: 'extent' },
-    { label: 'Subject', value: 'subjectLiteral', linkable: true },
-    { label: 'Genre/Form', value: 'materialType' },
-    { label: 'Notes', value: '' },
-    { label: 'Additional Resources', value: 'supplementaryContent', selfLinkable: true },
-    { label: 'Contents', value: 'note' },
-    { label: 'Bibliography', value: '' },
-    { label: 'ISBN', value: 'identifier', identifier: 'urn:isbn' },
-    { label: 'ISSN', value: 'identifier', identifier: 'urn:issn' },
-    { label: 'LCC', value: 'identifier', identifier: 'urn:lcc' },
-    { label: 'GPO', value: '' },
-    { label: 'Other Titles', value: '' },
-    { label: 'Owning Institutions', value: '' },
-  ];
-
-  const itemHoldings = items.length && !isElectronicResources ?
-    <ItemHoldings
-      shortenItems={shortenItems}
-      items={items}
-      bibId={bibId}
-      itemPage={itemPage}
-      searchKeywords={props.searchKeywords}
-    /> : null;
-  const marcRecord = isNYPLReCAP ? <MarcRecord bNumber={bNumber[0]} /> : null;
-
-  return (
-    <DocumentTitle title="Item Details | Shared Collection Catalog | NYPL">
-      <main className="main-page">
-        <div className="nypl-page-header">
-          <div className="nypl-full-width-wrapper">
-            <div className="nypl-row">
-              <div className="nypl-column-three-quarters">
-                <Breadcrumbs type="bib" query={searchURL} />
-                <h1>Item Details</h1>
-                <h2>{title}</h2>
-                {
-                  props.searchKeywords && (
-                    <div className="nypl-row search-control">
-                      <svg xmlns="http://www.w3.org/2000/svg" width="25" height="42" viewBox="0 0 25 42"
-                        preserveAspectRatio="xMidYMid meet" aria-labelledby="left-arrow" aria-hidden="true"
-                      >
-                        <title id="left-arrow">Back to Results</title>
-                        <polygon points="21.172 42.344 0 21.172 21.172 0 25.112 3.939 7.88 21.172 25.112 38.404 21.172 42.344"></polygon>
-                      </svg>
-                      <BackLink searchURL={searchURL} searchKeywords={props.searchKeywords} />
-                    </div>
-                  )
-                }
+
+  updateIsLoadingState(status) {
+    setTimeout(
```

Comment:
This isn't right. You're delaying _when_ the animation comes up but you should be delaying the _duration_ of the animation. If the duration is slightly longer, you'll see more of it and there'll be less of a flash.